mobile_files/DEPENDENCIES.md

```diff
@@ -0,0 +1,170 @@
+
+used to read QR code and take profile picture
+TODO: update version, this one doesn't work with Android (done in react-native upgrade branch)
+
+## "react-native-config": "git+https://github.com/status-im/react-native-config.git#0.11.2-1"
+
+used to fetch config values from env
+TODO: not sure fork usage is justified
+
+## "react-native-dialogs": "^1.0.5"
+
+used to show dialogs but only in android in list selection `status-im.ui.components.list-selection`, iOS uses action-sheet
+TODO: see if we should actually remove it
+
+## "react-native-fetch-polyfill": "^1.1.3"
+
+used to be able to put a timeout on fetch request
+TODO: check if the fetch timeout is actually aborting the request or only ignoring the callback on the response, if it
+doesn't then this is useless and we are better of using a simple seTimeout
+
+## "react-native-firebase": "5.1.1"
+
+used to have some stupid notifications and give free metadata to Google
+
+## "react-native-fs": "^2.14.1"
+
+used for some filesystem related functions, for instance to get no-backup directory in `status-im.utils.platform`
+
+## "react-native-gesture-handler": "1.3.0"
+
+library used by react-navigation
+use version recommended by react-navigation
+
+## "react-native-image-crop-picker": "^0.18.2"
+
+used for profile picture
+
+## "react-native-image-resizer": "git+https://github.com/status-im/react-native-image-resizer.git#1.0.##-status"
+
+used for profile picture
+
+## "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v.3.0.0-status"
+
+used for storing password when user saves password
+TODO: check if fork is necessary, merge with upstream?
+
+## "react-native-languages": "^3.0.2"
+
+TODO DEPRECATED, should be replaced with https://github.com/react-native-community/react-native-localize
+used to get user language and interact with i18n.js
+
+## "react-native-mail": "git+https://github.com/status-im/react-native-mail.git#v3.0.8_status"
+
+used to send email to support when phone is shaked
+
+## "react-native-navigation-twopane": "git+https://github.com/status-im/react-native-navigation-twopane.git#v0.0.2-status"
+
+used by desktop
+
+## "react-native-screens": "1.0.0-alpha.22"
+
+used by react-navigation
+
+TODO: update once bug is fixed for Android that causes crash when app is back from background after being put in background with hardware backbutton press
+
+## "react-native-shake": "^3.3.1"
+
+used to send email to support when phone is shaked
+## "react-native-splash-screen": "^3.2.0"
+
+used to make splash screen during initial loading
+
+## "react-native-status-keycard": "^2.5.7"
+
+used for keycard
+
+## "react-native-svg": "9.7.1"
+
+used for svg icons, mostly collectibles are using svg
+TODO: updating requires a pod update on iOS
+
+## "react-native-svg-transformer": "^0.12.1"
+
+TODO: not sure about this one, probably necessary, not used directly but used as a babel transform
```

Comment:
babel transform can be removed as well, isn't used

Reply:
what about collectibles?

Reply:
`React Native SVG transformer allows you to import SVG files in your React Native project ` but we don't import any svg files, we use webview to render svg collectables by url

Reply:
ok removed